forge-gui/res/languages/es-ES.properties

```diff
@@ -3299,4 +3299,4 @@ lblWraith=Fantasma
 lblWurm=Wurm
 lblYeti=Yeti
 lblZombie=Zombi
-lblZubera=Zubera
+lblZubera=Zubera
```

Comment:
there should be a newline at the end, right?

and the new prompt should still be in the file for each language... I just meant to create a new spot for PlaneswalkEffect prompts